marquez_airflow/dag.py

```diff
@@ -56,12 +58,20 @@ def __init__(self, *args, **kwargs):
             BigQueryOperator: BigQueryExtractor
             # Append new extractors here
         }
+        self.log.info(f'self._extractors: {self._extractors}')
+        self.log.info(f'self.args: {args}')
+        self.log.info(f'self.args: {kwargs}')
```

Comment:
- maybe put on one line? 
- second line is kwargs.
- is it different from the log on lines 44-47?